client/src/components/PartyContainer.js

```diff
@@ -27,13 +27,13 @@ const ContentContainer = styled.div`
 
 const FooterContainer = styled.div``;
 
-export default function PartyContainer({ button, ...props }) {
+export default function PartyContainer({ button, partyName, ...props }) {
   return (
     <>
       <BorderContainer>
         <HeaderContainer>
           <Header />
-          <PartyName />
+          <PartyName partyName={partyName.name} />
```

Comment:
`partyName.name` is confusing.
`partyName` already sounds like a String.

Reply:
Yes, I should rename it so it is clear that it is an object.

Reply:
It's named after the component should I still rename it?

Reply:
How does the object `partyName` looks like?
For me, it sounds like a `party`.

```suggestion
          <PartyName partyName={party.name} />
```